gpiozero/boards.py

```diff
@@ -1532,6 +1532,36 @@ def __init__(self, pwm=False, pin_factory=None):
         )
 
 
+class TrafficpHAT(TrafficLights):
+    """
+    Extends :class:`TrafficLights` for the `Pi Supply Traffic pHAT`_: Mini traffic
+    light LEDs.
+
+    The Traffic pHAT pins are fixed and therefore there's no need to specify
+    them when constructing this class. The following example then turns on all the LEDs::
+
+        from gpiozero import TrafficpHAT
+        hat = TrafficpHAT()
+        hat.lights.on()
```

Comment:
This is wrong now - should be `hat.on()` not `hat.lights.on()` and the instance should probably be called `phat` instead. May as well add an example like `phat.red.on()` too.